### DISCUSSION TITLE
Error : request Telegram API unsuccess, code error : 400

### DISCUSSION THREAD

**Discussion by danteistanto** [teamalgoritma/community#370] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/112854600/193264478-016d6269-ea0d-48b3-8381-bebfa746b22b.png)
Selamat malam, mohon bantuannya untuk error yang saya alami, terima kasih

**Comment by kevbow** [comment, marked as answer]
Halo @danteistanto,

Terdapat masalah ya dengan python interpreter yang digunakan. Silakan ctrl + shift + p lalu pilih environment khusus telebot dan jalankan ulang kembali bot_local.py.

Demikian dan terima kasih

**Reply by danteistanto** [reply]
terima kasih ka, sudah berhasil

**Comment by danteistanto** [comment]
> ![image](https://user-images.githubusercontent.com/112854600/193264478-016d6269-ea0d-48b3-8381-bebfa746b22b.png) Selamat malam, mohon bantuannya untuk error yang saya alami, terima kasih

solved : dibantu ka Kevin, problem di interpreter (View -> Command Palette (Ctrl+Shift+P) ). terima kasih
